<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>626</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TaskManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: white;
}

QTabBar::tab{
	color:black;
	background: transparent;
	border: none;
	padding-left: 15px;
	padding-right: 15px;
	padding-top: 3px;
	padding-bottom: 3px;
}

QTabBar::tab:selected {
	border-bottom : 3px solid rgb(51, 136, 255);
	padding-bottom: 3px;
}

QTabBar::tab:hover:!selected{
	color:rgb(190, 190, 190);
	padding-bottom: 3px;
}

QTabWidget::pane{
	border:none;
}

QTreeView::item:selected{
	background-color: rgb(205,232,255);
	color:black;
}

QTreeView::item:hover:!selected {
    background-color: rgb(229, 243, 255);
}

QTreeView::item{
	min-height:30px;
}

QTreeView::branch:selected{
	background:rgb(205,232,255);
}

QTreeView::branch:hover:!selected{
	background:rgb(229, 243, 255);
}

QHeaderView::section {
	min-height:50px;
	border:none;
	border-right:1px solid qlineargradient(spread:pad, x1:0.484, y1:0.994318, x2:0.489, y2:0.244, stop:0 rgba(165, 165, 165, 255), stop:1 rgba(255, 255, 255, 255));
	border-bottom:1px solid rgb(189, 189, 189);
	background-color:white;
}

QHeaderView::section:hover{
	background-color:rgb(229, 243, 255);
}

QListView{
	background:transparent;
	border:none;
}

QListView::item:selected{
	background-color: rgb(205,232,255);
	color:black;
	border:1px solid rgb(17, 125, 187);
}

QListView::item:hover:!selected {
   background-color: rgb(229, 243, 255);
	border:1px solid rgb(17, 125, 187);
}

QListView::item{
	min-height:80px;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="centralTabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="processTab">
       <attribute name="title">
        <string>Processes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeView" name="processView"/>
        </item>
        <item>
         <widget class="QWidget" name="killProcessWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>480</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QPushButton" name="killProcessButton">
             <property name="text">
              <string>Kill</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="performanceTab">
       <attribute name="title">
        <string>Performance</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="usageOptionList">
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
          <item>
           <property name="text">
            <string>CPU</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Memory</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="cpuPage">
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,6,6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="cpuLabel">
                <property name="text">
                 <string>CPU</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="cpuName">
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="UsagePlot" name="cpuUsagePLot" native="true"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1,1">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>40</number>
                </property>
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <property name="verticalSpacing">
                   <number>0</number>
                  </property>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Speed</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Utilization</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="processes">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="utilization">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLabel" name="temperature">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="speed">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>Processes</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>Temperature</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Up Time</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="upTime">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                 <number>15</number>
                </property>
                <property name="verticalSpacing">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Maximum Speed:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="maxSpeed">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Cores:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="cores">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Logical processors:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="logicalProcessors">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="memoryPage">
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,6,6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="memoryLabel">
                <property name="palette">
                 <palette>
                  <active/>
                  <inactive/>
                  <disabled/>
                 </palette>
                </property>
                <property name="text">
                 <string>Memory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="memoryName">
                <property name="palette">
                 <palette>
                  <active/>
                  <inactive/>
                  <disabled/>
                 </palette>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="UsagePlot" name="memoryUsagePlot" native="true"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>80</number>
                </property>
                <property name="horizontalSpacing">
                 <number>5</number>
                </property>
                <property name="verticalSpacing">
                 <number>0</number>
                </property>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_4">
                  <property name="palette">
                   <palette>
                    <active/>
                    <inactive/>
                    <disabled/>
                   </palette>
                  </property>
                  <property name="text">
                   <string>Available</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="usedLabel">
                  <property name="palette">
                   <palette>
                    <active/>
                    <inactive/>
                    <disabled/>
                   </palette>
                  </property>
                  <property name="text">
                   <string>Used</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="cached">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="usedMemory">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="reserved">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="availableMemory">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="cachedLabel">
                  <property name="palette">
                   <palette>
                    <active/>
                    <inactive/>
                    <disabled/>
                   </palette>
                  </property>
                  <property name="text">
                   <string>Cached</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="reservedLabel">
                  <property name="palette">
                   <palette>
                    <active/>
                    <inactive/>
                    <disabled/>
                   </palette>
                  </property>
                  <property name="text">
                   <string>Reserved</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="horizontalSpacing">
                 <number>10</number>
                </property>
                <property name="verticalSpacing">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="palette">
                   <palette>
                    <active/>
                    <inactive/>
                    <disabled/>
                   </palette>
                  </property>
                  <property name="text">
                   <string>Speed:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="palette">
                   <palette>
                    <active/>
                    <inactive/>
                    <disabled/>
                   </palette>
                  </property>
                  <property name="text">
                   <string>Sockets:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="palette">
                   <palette>
                    <active/>
                    <inactive/>
                    <disabled/>
                   </palette>
                  </property>
                  <property name="text">
                   <string>Reserved:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="memorySpeed">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="memorySockets">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="logicalProcessors_2">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_20">
                  <property name="palette">
                   <palette>
                    <active/>
                    <inactive/>
                    <disabled/>
                   </palette>
                  </property>
                  <property name="text">
                   <string>Reserved:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="temperature_2">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UsagePlot</class>
   <extends>QWidget</extends>
   <header location="global">usageplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>centralTabWidget</tabstop>
  <tabstop>processView</tabstop>
  <tabstop>killProcessButton</tabstop>
  <tabstop>usageOptionList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
